### PR TITLE
Added MMA2 tick sound as an option for note sound.

### PR DESCRIPTION
**Pull request by Brollyy**
Credit to https://git.bsmg.dev/Top_Cat/MediocreMapAssistant2 for the sound file.

### CODE REVIEW

File: Classes/Audio/ParallelAudioPlayer.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public class ParallelAudioPlayer: IDisposable {
                 Volume = defaultVolume
             };
             notePlayers[i] = new WasapiOut(AudioClientShareMode.Shared, desiredLatency);
-            if (isPanned) {
+            if (isPanned && basePath != "mmatick") {
```
Comment (PKBeam) [inline Classes/Audio/ParallelAudioPlayer.cs:49]: what's the reason for this change?

Comment (Brollyy) [reply Classes/Audio/ParallelAudioPlayer.cs:49]: The WAV they're using in MMA2 only has one channel, so StereoToMonoSampleProvider throws an exception when the note sound panning is enabled.

**Review by PKBeam — APPROVED** [review]
